bin/fzf-tmux

```diff
@@ -7,8 +7,8 @@ fail() {
   exit 2
 }
 
-fzf="$(command -v fzf 2> /dev/null)" || fzf="$(dirname "$0")/fzf"
-[[ -x "$fzf" ]] || fail 'fzf executable not found'
+fzf=$(command -v fzf 2>/dev/null) || fzf="$(dirname "$0")/fzf"
```

Comment:
Why do we remove the space before `/dev/null`?

Reply:
Related? https://github.com/koalaman/shellcheck/issues/1344#issuecomment-443542106

Reply:
Was just a stylistic choice, I removed it now.